TWLight/emails/templates/emails/contact_us_email-subject.html

```diff
@@ -0,0 +1,5 @@
+{% load i18n %}
+{% comment %} Translators: This is the subject line of an email sent to the Library Card platform admins from the contact us form.{% endcomment %}
+{% blocktrans trimmed %}
+Library Card platform email from user "{{ editor_wp_username }}"

```

Comment:
I manually tested usernames with utf-8 characters since a) those are allowed in wp usernames and b) they can be problematic in email. It looks like Djmail does the right thing on our behalf, or at least doesn't fall over and die. Let's change this subject line up a bit though:
```suggestion
Wikipedia Library Card Platform message from {{ editor_wp_username }}
```